src/tools/scripts/incoherenceStructure/structure_sans_coselec.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../../utils');
+const path = require('path');
+const fs = require('fs');
+
+const getStructures = async db => await db.collection('structures').find({
+  'statut': { '$ne': 'VALIDATION_COSELEC' }
+}).toArray();
+
+const countUserStructure = db => async idStructure => await db.collection('users').countDocuments({
+  'entity.$id': idStructure
+});
+
+const countStatutsIncorrects = db => async idStructure => await db.collection('misesEnRelation').aggregate(
+  { '$match': {
+    'structure.$id': idStructure,
+    'statut': { '$ne': 'finalisee_rupture' }
```

Comment:
`$nin: ['finalisee_rupture', 'terminee']`  (peut y avoir si on réutilise le script dans un futur proche)

Reply:
ah yes : limite on pourrait faire sauter ce $in pour avoir les stats de finalisee rupture et terminee dans le log final

Reply:
On était sur un not equal, mais ok on ne filtre plus :)